packages/smooth_app/lib/pages/personalized_ranking_page.dart

```diff
@@ -67,125 +112,28 @@ class _PersonalizedRankingPageState extends State<PersonalizedRankingPage> {
-                            context: context,
-                            backgroundColor: Colors.transparent,
-                            bounce: true,
-                            barrierColor: Colors.black45,
-                            builder: (BuildContext context,
-                                    ScrollController scrollController) =>
-                                UserPreferencesView(scrollController,
-                                    callback: () {
-                              personalizedRankingModel.processProductList();
-                              const SnackBar snackBar = SnackBar(
-                                content: Text(
-                                  'Reloaded with new preferences',
-                                ),
-                                duration: Duration(milliseconds: 1500),
-                              );
-                              personalizedRankingModel.scaffoldKey.currentState
-                                  .showSnackBar(snackBar);
-                            }),
-                          ),
-                        ),
-                      ),
-                    ],
-                  ),
-                ),
-              ],
-            )*/
+      body: _buildGroupedStickyListView(),
     );
   }
 
-  Widget _buildSmoothProductCard(
-    RankedProduct rankedProduct,
-    BuildContext context,
-    Color backgroundColor,
-  ) =>
-      Padding(
+  Widget _buildSmoothProductCard(final RankedProduct rankedProduct) => Padding(
         padding: const EdgeInsets.symmetric(horizontal: 12.0, vertical: 8.0),
-        child: Column(
-          children: <Widget>[
-            //Text(rankedProduct.score.toString()),
-            SmoothProductCardFound(
-              heroTag: rankedProduct.product.barcode,
-              product: rankedProduct.product,
-              elevation: 4.0,
-              backgroundColor: backgroundColor,
-            ).build(
-                context), // TODO(monsieurtanuki): check that "build.content" thing
-          ],
+        child: SmoothProductCardFound(
+          heroTag: rankedProduct.product.barcode,
+          product: rankedProduct.product,
+          elevation: 4.0,
+          backgroundColor: _COLORS[SmoothItModel.getMatchIndex(rankedProduct)],
         ),
       );
 
-  Widget _buildGroupedStickyListView(BuildContext context) => AnimatedContainer(
+  Widget _buildGroupedStickyListView() => AnimatedContainer(
         duration: const Duration(milliseconds: 200),
-        padding: EdgeInsets.only(
-            top: _showTitle ? MediaQuery.of(context).size.height * 0.0 : 0.0),
         child: CustomScrollView(
           controller: _scrollController,
           slivers: <Widget>[
             SliverAppBar(
               leading: IconButton(
-                icon: Icon(Icons.arrow_back, color: Theme.of(context).accentColor,),
+                icon: Icon(ConstantIcons.getBackIcon(), Theme.of(context).accentColor),
```

Comment:
I suspect this gives an error
```suggestion
                icon: Icon(ConstantIcons.getBackIcon(), color: Theme.of(context).accentColor),
```